qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -385,44 +398,44 @@ def job(
 
     @requires_submit
     def qobjs(self) -> List[Qobj]:
-        """Return the Qobj for the jobs.
+        """Return the Qobj's for the jobs in this set.
 
         Returns:
-            A list of Qobj for the jobs. The entry is ``None`` if the Qobj
-                could not be retrieved.
+            A list of Qobj's for the jobs. The entry is ``None`` if the Qobj
+            could not be retrieved.
```

Comment:
Might help to consolidate this sentence, something like: `A list of Qobjs, for the jobs in the set, or ``None`` if the Qobjs could not be retrieved`?
It is an interesting sentence because of all the words ending is `s` 😆

Reply:
Your sentence sounds to me that the return value is ``None`` if Qobjs couldn't be retrieved. What I'm trying to say, however, is that _an entry_ in the list is ``None`` if Qobj for that particular job could not be retrieved. Updated with
`A list of ``Qobj``s for the jobs. An entry in the list is ``None`` if the ``Qobj`` for that job could not be retrieved.`

Reply:
I misunderstood, but the wording of the last sentence makes everything much more clear that it is the specific entries in the list that may be None.